warehouse/templates/pages/help.html

```diff
@@ -152,6 +154,17 @@ <h2>{{ project_name_claim() }}</h2>
         </p>
       </section>
 
+      <section id="file_size_limit" class="common-question">
+        <h2>{{ file_size_limit() }}</h2>
+        <p>
+          If you can't upload your release to PyPI because you're hitting the upload file size limit, please <a href="https://github.com/pypa/warehouse/issues/new">file an issue</a> and tell us:</p>
+        <ul>
+          <li>The name of the project</li>
+          <li>The size of your release (example: 300 MB)</li>
+          <li>Which index/indexes you need the increase for (PyPI, Test PyPI, or both)</li>
```

Comment:
It's probably also worth adding somewhere (perhaps here) that if they are trying to upload the _first_ release for their project, and that release is larger than the default limit, they need to make an initial release that is smaller than the default liit before we are able to increase the limit for the given name.

Reply:
Thanks. Fixed.